device_doctor/lib/src/device.dart

```diff
@@ -39,11 +53,6 @@ abstract class Device {
   /// Whether the device is asleep.
   Future<bool> isAsleep();
```

Comment:
is this and the method above supported in ios?

Reply:
Removed.